source/_integrations/madvr.markdown

```diff
@@ -62,9 +64,11 @@ These are commands that have parameters with a comma separating them.
 - `KeyPress (MENU | UP | DOWN | LEFT | RIGHT | OK | INPUT | SETTINGS | RED | GREEN | BLUE | YELLOW | POWER)`
 - `KeyHold (MENU | UP | DOWN | LEFT | RIGHT | OK | INPUT | SETTINGS | RED | GREEN | BLUE | YELLOW | POWER)`
 
-### Binary sensor
+### Usage Example
```

Comment:
```suggestion
### Usage example
```
We use [sentence-style capitalization](https://learn.microsoft.com/en-us/style-guide/text-formatting/using-type/use-sentence-style-capitalization), also in headings